_posts/2016-12-06-future.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: post
+title:  "The State of Babel"
+author: Henry Zhu
+date:   2016-12-06 11:00:00
+categories: announcements
+share_text: "The State of Babel"
+---
+
+- Some History
+- Current Status
+- Future
+- Community
+
+> Previous issues: [Babel Roadmap #4130](https://github.com/babel/babel/issues/4130), [6.0 #2168](https://github.com/babel/babel/issues/2168)
+
+## Some History
+
+[Sebastian](https://github.com/kittens) created Babel in September of 2015. Interestingly enough, he made it to scratch an itch he had with understanding programming languages and how they work. You might of assumed whoever created the project already knew how compilers worked and understood JavaScript perfectly! Check out his post for a great read of his story: [~2015 in Review](https://medium.com/@sebmck/2015-in-review-51ac7035e272#.jdoo279bl).
+
+When 6to5 became Babel as mentioned in [Not Born to Die](https://babeljs.io/blog/2015/02/15/not-born-to-die), it became a platform: a suite of tools designed to create the next generation of JavaScript tooling. No longer did it just compile ES6 to ES5, it allowed developers to build tools on top of it.
```

Comment:
Nit: Make these relative links (`/blog/2015/.....`) so they go to the right place when testing the site locally?